config/index.js

```diff
@@ -5,67 +5,152 @@ module.exports = {
   },
   rules: {
     'accessor-pairs': 'error',
+    'array-bracket-newline': 'off',
+    'array-bracket-spacing': ['error', 'never'],
+    'array-callback-return': 'error',
+    'array-element-newline': 'off',
+    'arrow-body-style': 'off',
+    'arrow-parens': 'error',
     'arrow-spacing': ['error', { 'before': true, 'after': true }],
+    'block-scoped-var': 'error',
     'block-spacing': ['error', 'always'],
     'brace-style': 'error',
+    'callback-return': 'error',
     'camelcase': ['error', { 'properties': 'never', 'allow': ['^UNSAFE_'] }],
+    'capitalized-comments': 'off',
+    'class-methods-use-this': 'off',
     'comma-dangle': ['error', 'always-multiline'],
     'comma-spacing': ['error', { 'before': false, 'after': true }],
     'comma-style': ['error', 'last'],
+    'complexity': 'off',
+    'computed-property-spacing': 'error',
+    'consistent-return': 'error',
+    'consistent-this': ['error', 'self'],
     'constructor-super': 'error',
     'curly': 'error',
     'default-case': 'error',
+    'default-param-last': 'error',
     'dot-location': ['error', 'property'],
+    'dot-notation': 'error',
     'eol-last': 'error',
     'eqeqeq': ['error', 'allow-null'],
+    'for-direction': 'error',
+    'func-call-spacing': 'error',
+    'func-name-matching': 'error',
+    'func-names': 'off',
+    'func-style': 'off',
+    'function-call-argument-newline': 'off',
+    'function-paren-newline': ['error', 'consistent'],
     'generator-star-spacing': ['error', { 'before': true, 'after': true }],
-    'handle-callback-err': ['error', '^(err|error)$' ],
+    'getter-return': 'error',
+    'global-require': 'error',
+    'grouped-accessor-pairs': 'error',
+    'guard-for-in': 'error',
+    'handle-callback-err': ['error', '^(err|error)$'],
+    'id-blacklist': 'off',
+    'id-length': 'off',
+    'id-match': 'off',
+    'implicit-arrow-linebreak': 'error',
     'indent': ['error', 2, { 'SwitchCase': 1 }],
+    'init-declarations': 'off',
     'jsx-quotes': ['error', 'prefer-double'],
     'key-spacing': 'error',
     'keyword-spacing': ['error', { 'before': true, 'after': true }],
+    'line-comment-position': 'off',
+    'linebreak-style': 'error',
+    'lines-around-comment': 'error',
+    'lines-around-directive': 'error',
```

Comment:
This rule has been deprecated, and replaced by `padding-line-between-statements`

Reply:
Likewise for `newline-after-var` and `newline-before-return`

Reply:
Disabled this one, the other two are already off.

Reply:
Oh - I didn't realize you were keeping them around intentionally. I would have omitted them. I guess it doesn't matter though.

Reply:
I like the deprecated comments you added - good solution